config/deploy.rb

```diff
@@ -93,6 +93,10 @@
       default_run_options[:shell] = '/bin/bash --login'
       run "god -c /var/www/blog/monitoring/keepalive.god"
     end
+    task :shim, roles: :app do
+      default_run_options[:shell] = '/bin/bash --login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.